salt/modules/win_ip.py

```diff
@@ -319,13 +319,35 @@ def set_static_dns(iface, *addrs):
     '''
     Set static DNS configuration on a Windows NIC
 
+    Args:
+
+        iface (str): The name of the interface to set
+
+        addrs (*):
+            One or more DNS servers to be added
+
+            .. note::
```

Comment:
This does not seem like the right behavior to me. These sorts of side-effects are implicit and, in my opinion, should be explicit. IMHO, nothing passed should mean no change is applied and a list should only be cleared if, and only if, an empty list is explicitly passed in.